liblangutil/Scanner.cpp

```diff
@@ -343,13 +366,13 @@ Token Scanner::scanMultiLineDocComment()
 	bool endFound = false;
 	bool charsAdded = false;
 
-	while (isWhiteSpace(m_char) && !isLineTerminator(m_char))
+	while (isWhiteSpace(m_char) && !isLineTerminator())
```

Comment:
This loop will terminate if there is a single `\r` not followed by `\n`. Is that on purpose?

Reply:
Sorry: It will consume the `\r` that is not followed by `\n`.